containers/proxy-squid-image/squid.conf

```diff
@@ -5,13 +5,15 @@
 http_port 8080
 
 cache_mem 400 MB
-maximum_object_size 200 MB
+
+# cached images can be large
+maximum_object_size 10 GB
 maximum_object_size_in_memory 1024 KB
 
 access_log /var/log/squid/access.log squid
 
 # Size should be about 60% of your free space
-cache_dir ufs /var/cache/squid 15000 16 256
+cache_dir aufs /var/cache/squid 15000 16 256
```

Comment:
Could you please adapt this change [here](https://github.com/uyuni-project/uyuni/blob/master/containers/proxy-squid-image/uyuni-configure.py#L14) as well? Otherwise the `uyuni-configure.py` will not do the right thing here. Thank you.

Reply:
Done. Btw. I did this change because of what is recommended in [SUSE squid admin guide](https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-squid.html#sec-squid-configfile-general)